modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/objectViewContext.tsx

```diff
@@ -313,7 +326,10 @@ const viewReducer = (state: TState, action: TAction) => {
 						...filterColumn,
 						definition,
 						fieldLabel: objectField
-							? objectField.label[defaultLanguageId]
+							? getLocalizableLabel(
+									objectField.label,
+									creationLanguageId
+							  )
```

Comment:
Hey @alinedoleron, you can use the `objectField.name` as the `fallback` of the function